.eslintrc

```diff
@@ -10,6 +10,11 @@
     "prettier",
     "simple-import-sort"
   ],
+  "parser": "@typescript-eslint/parser",
```

Comment:
Interesting, this one is already present as a dependency of `@typescript-eslint/eslint-plugin` and docs recommend to explicitly add it as a parser. Looks like something we actually needed for some time already!

```shell
└─┬ react-scripts@5.0.0
  └─┬ eslint-config-react-app@7.0.0
    ├─┬ @typescript-eslint/eslint-plugin@5.14.0
    │ └── @typescript-eslint/parser@5.14.0 deduped
    └── @typescript-eslint/parser@5.14.0
```

Reply:
For sure, I think we are strictly better off with it.